docs/getting-started-step-by-step/00-overview.md

```diff
@@ -0,0 +1,44 @@
+# Overview Getting Started: Step-by-Step
+
+The following guide provides an easy, step-by-step path for developers and platform engineers to get familiar with Crossplane. It complements the current Getting Started guide, which focuses on getting an RDS Instance up and running quickly. In this example will use Object Storage as a simple example with quick turn around times.
```

Comment:
I wonder whether this guide (or somewhere adjacent?) would be a good place to frame up Crossplane's approach to infrastructure - e.g. the separation of concerns and interactions between platform teams and app teams. My hunch is that some of our complexities might make more sense to the reader with that context preloaded.

Reply:
Good call. I agree we need to bind this to our approach and its value. I've [rephrased it](https://github.com/luebken/crossplane/pull/1/commits/d9b80835171e43625cac563e307fc11036e95386) and added a link in the "More Info" section: https://github.com/luebken/crossplane/pull/1/commits/721f4ebd21b5bd071cce860a4c42df8906577db1

@negz WDYT?

Reply:
> This guide complements the current Getting Started guide, which focuses on getting an RDS Instance up and running quickly.

Should we really have two getting started guides? I'd prefer to replace/refactor/improve the existing one, as opposed to for example having two different "how to install Crossplane" pages.

> WDYT

One approach here would be to literally break out the guide into the parts that would be done by the platform team (e.g. writing the XRD and compositions) and the parts that would be done by the application developers (e.g. authoring claims). This might be a little heavy handed here though; I imagine the persona you're targeting with this guide is a platform engineer who is interested in learning Crossplane, right? If so, I would consider a preface calling out explicitly the fact that in "real world" use cases different parts of this process would be done by different people. This way the platform engineer understands why there are different layers where perhaps there would not need to be if they were the only persona using Crossplane.